src/mongo/MongoScrapbook.ts

```diff
@@ -204,4 +207,46 @@ class MongoScriptDocumentVisitor extends MongoVisitor<MongoCommand[]> {
 	protected defaultResult(_node: ParseTree): MongoCommand[] {
 		return this.commands;
 	}
+
+	private parseContext(ctx: mongoParser.ArgumentContext | mongoParser.PropertyValueContext): Object {
```

Comment:
Better name.  argumentToObject?

Reply:
I tried to stay away from having argument in the name since it could be an argument or Property value.

Reply:
contextToObject?